app-arch/snappy/snappy-1.1.9.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib
+
+DESCRIPTION="A high-speed compression/decompression library by Google"
+HOMEPAGE="https://github.com/google/snappy"
+SRC_URI="https://github.com/google/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV%%.*}"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="test? ( dev-cpp/gtest )"
```

Comment:
`DEPEND` was correct here (as is in `snappy-1.1.8.ebuild`).
`dev-cpp/gtest` contains only headers and libraries, not executables.

Reply:
gtest is not runtime dependency, snappy does not require it for working, it's buildtime dependency.

Reply:
Both `BDEPEND` and `DEPEND` are for build-time dependencies, needed in different locations. See e.g. [this](https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#improved-cross-compilation-support).
`RDEPEND` is for run-time dependencies.

`dev-cpp/gtest` should be in `DEPEND`.